config/config.go

```diff
@@ -46,6 +50,11 @@ const (
 	defaultConfigJson string = `
 {
 	"auto_ssl_cache_dir": ".certs",
+    "ipfs": {
```

Comment:
I would suggest replace it by string "http://loclahost:5051" to simplify processing